manpages/lumina-archiver.8

```diff
@@ -0,0 +1,38 @@
+.Dd October 24, 2017
+.Dt LUMINA-ARCHIVER 8
+.Os 1.3.3
+
+.Sh NAME
+.Nm lumina-archiver
+.Nd is a graphical front-end to tar and is used to manage and/or create archives.
+
+.Sh SYNOPSIS
+.Nm
+.Op <argument> 
+.Op <file>
```

Comment:
I don't think the .Op <argument>/ .Op <file> syntax is correct, but I'll review again when I'm back in town with all my notes.